package.json

```diff
@@ -30,7 +30,7 @@
     "start": "npm run webpack -- --progress --watch",
     "test": "npm run format:check && npm run test:unit && npm run jest-ssr && npm run karma && npm run jest-screenshot",
     "test:unit": "vitest run",
-    "percy-screenshot": "babel-node ./test/percy/server/createButtonConfigs.js && percy exec -- playwright test --config=./test/percy/playwright.config.js --reporter=dot --pass-with-no-tests",
+    "percy-screenshot": "npx playwright install && babel-node ./test/percy/server/createButtonConfigs.js && percy exec -- playwright test --config=./test/percy/playwright.config.js --reporter=dot --pass-with-no-tests",
```

Comment:
do we still need this if we updated percy and playwright in the package.json?

Reply:
yes! I only know this because the second commit in this PR upgraded the dependencies and still failed: https://github.com/paypal/paypal-checkout-components/actions/runs/6397759802/job/17366408426 but the third commit added this line and passed 🥳 

Another option would be to back out the dependency upgrades and only keep `npx playwright install`.